source/assets/stylesheets/base/_forms.scss

```diff
@@ -1,3 +1,7 @@
+form {
+  @include clearfix;
+}
+
 fieldset {
   background: lighten($base-border-color, 10);
```

Comment:
Thanks. Skipping this for now because it came with bitters, and was unchanged in this commit. I'd want to be thorough for consistency.